### ISSUE TITLE
Running run_def_tests.cmd left-pad on windows does not process any type definitions

### ISSUE THREAD

**Issue by doberkofler** [flowtype/flow-typed#735]
Independently of running with or without arguments, `run_def_tests.cmd` does not seem to process any files in the final test suite as done by Travis

```
D:\flow-typed\cli>node dist\cli.js validate-defs ..\definitions
All libdefs are named and structured correctly. (Found 225)
D:\flow-typed\cli>node dist\cli.js run-tests selenium-webdriver
Running definition tests in D:\flow-typed\definitions\npm...
Fetching all Flow binaries...
All tests passed!
D:\flow-typed\cli>cd ..
```

**Comment by jeffmo** [comment]
FYI: I just published `v2.1.0` which contains this fix

**Comment by doberkofler** [comment]
Solved!
